src/App.js

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 
+const todoList = [
+  {
+    id:1,
+    title:"Complete assignment"},
+  {
+    id:2,
+    title:"Make an appointment"
+  },
+  {
+    id:3,
+    title:"Call mom"
+  }
+]; 
+
 function App() {
+  const listItems = todoList.map(function(item){
+    return <li>{item.id} {item.title}</li>;
+  });
```

Comment:
item.id should be used as key, and item text should only contain title. 

doc in the link has some examples for how to setting key in li element https://legacy.reactjs.org/docs/lists-and-keys.html